src/sentry/sentry_metrics/querying/data/execution.py

```diff
@@ -445,12 +447,24 @@ def modified_end(self) -> datetime:
 
     @property
     def series(self) -> Sequence[Mapping[str, Any]]:
+        if "series" not in self.result:
+            return []
```

Comment:
In this case, we have to check for series not being available in the result, because otherwise we would have to catch the error (not nice); something like hasattr() also does not work, because it actually calls the attribute to see if it exists.